C/CImGui/build_tarballs.jl

```diff
@@ -44,4 +45,4 @@ dependencies = Dependency[
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies, julia_compat="1.7")
```

Comment:
Why 1.7 and not 1.6? Also, I guess we may want to do the same for v1.82?

Reply:
I thought the point was that 1.6 couldn't handle experimental platforms? Maybe I misunderstood Max's comment about "after 1.6"

> after 1.6 those checks were improved/relaxed so that we can introduced further platforms w/o breaking things, but this issues was not recognized as such before the attempt to add a new platform (Apple Silicon) and things going boom